src/App.js

```diff
@@ -1,25 +1,51 @@
-import logo from './logo.svg';
+import {
+  Link, Route, Routes, useLocation,
+} from 'react-router-dom';
+import { useEffect } from 'react';
+import { Navbar } from 'react-bootstrap';
+import Details from './components/Details/Details';
+import Home from './components/Home/Home';
 import './App.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+import music from './assets/110-pokemon_center.mp3';
 
-function App() {
+const App = () => {
+  const { pathname } = useLocation();
+  useEffect(() => {
+    // window.scrollTo(0, 0);
+  }, [pathname]);
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
+    <>
+      <div className="container-fluid min-width">
+        <audio /* eslint-disable-line */
```

Comment:
- [ ] You have **disabled linter check** on your code. Please enable linter check on your code to ensure that your code is robust code and clean.

Reply:
Thanks for your kindly and useful feedback @rloterh. :+1: :1st_place_medal:  I implement the solution for this in the commit a3ca22e